src/pages/_app.tsx

```diff
@@ -101,11 +102,17 @@ function AppContent({ Component, pageProps }: AppProps<AppCommonProps>) {
             </EvmProvider>
           </ErrorBoundary>
         </div>
+        <PostSubscriber />
       </QueryProvider>
     </ThemeProvider>
   )
 }
 
+function PostSubscriber() {
```

Comment:
Strange component. Can we just use this hook on line 74?

Reply:
I have to add this because this hook needs to be inside of query client provider, so I can't just call it inside _app